pkgs/top-level/python-aliases.nix

```diff
@@ -77,6 +77,7 @@ mapAliases ({
   bt_proximity = bt-proximity; # added 2021-07-02
   BTrees = btrees; # added 2023-02-19
   carrot = throw "carrot has been removed, as its development was discontinued in 2012"; # added 2022-01-18
+  cacheyou = throw "cacheyou has been removed, as it was no longer used for the only consumer pdm"; # added 2023-12-21
```

Comment:
cac sort before car ?

Reply:
Haha, looks like the dictionary I used was wrong.